content/docs/command-reference/fetch.md

```diff
@@ -42,20 +42,22 @@ project's cache                  ++ | dvc pull |
  workspace
 ```
 
-Fetching could be useful when first checking out a <abbr>DVC project</abbr>,
-since files tracked by DVC should already exist in remote storage, but won't be
-in the project's <abbr>cache</abbr>. (Refer to `dvc remote` for more information
-on DVC remotes.) These necessary data or model files are listed as
-<abbr>dependencies</abbr> or <abbr>outputs</abbr> in a target
-[stage](/doc/command-reference/run) (in `dvc.yaml`) or `.dvc` file, so they are
-required to [reproduce](/doc/tutorials/get-started/data-pipelines#reproduce) the
-corresponding [pipeline](/doc/command-reference/pipeline).
-
-`dvc fetch` ensures that the files needed for a stage or `.dvc` file to be
-[reproduced](/doc/tutorials/get-started/data-pipelines#reproduce) exist in
-cache. If no `targets` are specified, the set of data files to fetch is
-determined by analyzing all `dvc.yaml` and `.dvc` files in the current branch,
-unless `--all-branches` or `--all-tags` is specified.
+Fetching is useful when first checking out a <abbr>DVC project</abbr> for
+example, to get any files tracked by DVC that already exist in remote storage
+(see `dvc push`) to the local <abbr>cache</abbr>. Refer to `dvc remote` for more
+information on DVC remotes.
+
+`dvc fetch` ensures that the files needed for all the stages (in
+[`dvc.yaml`](/doc/user-guide/dvc-files-and-directories#dvcyaml-file)) and
+[`.dvc` files](/doc/user-guide/dvc-files-and-directories#dvc-files) exist in the
+cache. It can be limited to any given `targets` (files inside directories
```

Comment:
can we unify the approach with other commands? It looks like we use different variations here and there

Reply:
`targets` can be specified to download only files that are needed. It accepts either `.dvc` files, or stage names from `dvc.yaml`, or any regular path to a DVC-tracked file, see examples ..

Reply:
> It looks like we use different variations here and there

Because some use `targets` and some use `path` so it's a little different. The ones that use targets didn't really have a paragraph explaining that argument so the text I put in them about this may be a little different. I'll standardize now, similar to checkout (using an individual paragraph about targets).

Reply:
Unified this to what we just did with checkout (introduce paragraph about targets arg near the top) in 73512e17.

Reply:
P/s your suggested text is nice and short but it didn't mention the granularity part.

Reply:
Applied to status too now (in b89fed8).

Reply:
To apply this to push and pull though, it will require significant rephrasing and some reordering of the Description paragraphs, so I'll open a separate PR, along with other out-of-scope changes I've stashed while working on the above.

Reply:
> To apply this to push and pull

I put in the unified notes for push and pull here in 20927fe6 🙂 

Opened #1602 for everything else.